src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const isValidTabId = tabs.some(tab => tab.id === selectedTabId);
+
+  if (!isValidTabId) {
+    // eslint-disable-next-line no-param-reassign
+    selectedTabId = tabs[0].id;
+  }
```

Comment:
do not reassign props.
insteas you can make something like that.
and use selectedTab.content for render content
```suggestion
   const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```